documentation/pages/product1/photon-overview.md

```diff
@@ -0,0 +1,5 @@
+# Docker volumes on Photon platform
+
+Docker volumes on [Photon platform](https://vmware.github.io/photon-controller) are provisioned, managed entirely via the Photon platform [API](https://github.com/vmware/photon-controller/wiki/API). Docker volumes are placed on datastores as decided by the Photon platform. The Photon platform selects datastores from within those that have been explicitly configured by the user on hosts managed by the Photon Controller and include backends such as NFS, SAN, VSAN, vVol. Accessibility of volumes to VMs running on the hosts is therefore dependent on whether the datastores are shared among hosts configured to the Photon platform. Accessibility of docker volumes on Photon is governed by the tenancy model of Photon, where each volume is scoped to a project within a tenant. Volumes are scoped to a project and hence accessible to all VMs within the same project. In addition, the count of volumes and storage capacity allocated (in total) to docker volumes within a project are managed via resource limits defined by the resource ticket associated with the tenant that the project belongs to. The Photon platform admin must hence take care to configure sufficient storage capacity, taking into consideration the max. number of volumes and the typical sizes of those. Once a configuration has been created, the Photon platform seamlessly manages accounting the count and storage capacity consumed by volumes in a project.

```

Comment:
@govint Thanks for your comment. But could you please elaborate ? I also checked this section and it was added six months back by you https://goo.gl/VaxfwI. So it would be great if you could provide me a rephrased version. Thanks